mix.exs

```diff
@@ -21,15 +21,24 @@ defmodule BroadwayRabbitMQ.MixProject do
   end
 
   def application do
-    [
-      extra_applications: [:lager, :logger]
-    ]
```

Comment:
I think we can just remove :lager altogether, no? Do things fail in earlier versions if do we need to list it here? 🤔

Reply:
Oh that's a good point, amqp 1.x/2.x already had lager in its extra_applications! So yes this works fine without it.